tests/Spout/Writer/XLSX/WriterTest.php

```diff
@@ -320,6 +344,7 @@ public function testAddRowShouldSupportAssociativeArrays()
      */
     public function testAddRowShouldNotWriteEmptyRows()
     {
+        $this->markTestIncomplete('Unsure why this does not pass');
```

Comment:
Actually, this is caused by a condition change is AbstractWriter. `!empty($dataRow)` is different from `$row->isEmpty()`. The former returns false for `['']` while the latter returns true.